pandas/core/groupby.py

```diff
@@ -2315,6 +2315,11 @@ def __init__(self, index, grouper=None, obj=None, name=None, level=None,
                 # groupby
                 else:
                     cat = self.grouper.unique()
```

Comment:
`self.grouper` can also be a `CategoricalIndex`. How do I get one from the other?